mappings/13w01b#13w02a.tinydiff

```diff
@@ -1712,14 +1712,14 @@ c	net/minecraft/unmapped/C_6607881
 		p	2			stage
 	m	()Lnet/minecraft/unmapped/C_6607881;	m_1048888		getLastAttacker
 	m	()Ljava/lang/String;	m_0699887		m_0699887
-	m	(Ljava/lang/String;)V	m_3910252		m_3910252
-		p	1			p_1
-	m	()Ljava/lang/String;	m_3890403		m_3890403
-	m	()Z	m_4688620		m_4688620
-	m	(Z)V	m_7191334		m_7191334
-		p	1			p_1
-	m	()Z	m_6295227		m_6295227
-	m	()Z	m_3400324		m_3400324
+	m	(Ljava/lang/String;)V	m_3910252		setCustomName
+		p	1			name
+	m	()Ljava/lang/String;	m_3890403		getCustomName
+	m	()Z	m_4688620		customNameExists
```

Comment:
Just going off of the name here but there are other methods we've called `hasCustomName` that by the sound of it do a similar thing, so perhaps one or the other should be changed to keep it consistent?